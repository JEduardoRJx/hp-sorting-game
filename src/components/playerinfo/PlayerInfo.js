import React from 'react';
import './PlayerInfo.scss'

const PlayerInfo = () => {
  return(
    <aside>
      <img className='crest' 
        src={require('../../images/hogwarts_crest.png')}
        alt='hogwarts crest'/>
    </aside>
  )
}

export default PlayerInfo