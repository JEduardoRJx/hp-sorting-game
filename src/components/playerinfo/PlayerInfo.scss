$hw-dark: #4c3638;
$hw-light: #D3A481;
$sl-dark: #276F45;
$sl-light: #A8B0BF;
$gryf-dark: #702737;
$gryf-light: #DAA520;
$rav-dark: #274F70;
$rav-light: #C98C4E;
$huff-dark: #141414;
$huff-light: #D1B738;

aside {
  width: 100%;
  background-color: $hw-light;
  color: $hw-dark;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: space-around;
  &:gryffindor {
  background-color: $gryf-light;
  }
}

.crest {
  height: 350px;
}

.user-name {
  font-size: 2.5em;
  text-align: center;
}

.user-wizardkind,
.user-wand {
  font-size: 1.75em;
  text-align: center;
}