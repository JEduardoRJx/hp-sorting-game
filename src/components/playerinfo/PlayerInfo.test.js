import React from 'react';
import { shallow } from 'enzyme';
import { PlayerInfo } from './PlayerInfo'

describe('PlayerInfo', () => {
  let wrapper;
  const mockUser = {
    name: 'name',
    wizardkind: 'wizard',
    wand: 'wand',
    house: 'slytherin'
  }

  beforeEach(() => {
    wrapper = shallow(<PlayerInfo 
    user={mockUser} />)
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })

  it.skip('should match user variables', () => {
    //Need to test line 6, 7, 8, & 9 of PlayerInfoComponent
  })
})