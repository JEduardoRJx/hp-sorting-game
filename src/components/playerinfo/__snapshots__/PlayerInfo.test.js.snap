// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerInfo should match snapshot 1`] = `
<aside
  className="slytherin"
>
  <img
    alt="hogwarts crest"
    className="crest"
    src="slytherin.png"
  />
  <h1
    className="user-name"
  >
    name
  </h1>
  <h2
    className="user-wizardkind"
  >
    wizard
  </h2>
  <h2
    className="user-wand"
  >
    wand
  </h2>
</aside>
`;
